compiler/src/iree/compiler/Codegen/Common/TileDispatchUsingForall.cpp

```diff
@@ -294,6 +340,7 @@ void TileAndDistributeToWorkgroupsUsingForallOpPass::runOnOperation() {
 
   // If the `tilableOp` is a `memref` op, then just tile the operation.
   SmallVector<LoopLikeOpInterface> tilingLoops;
+  Operation *lastTiledProducerForConsumerFusion = nullptr;
```

Comment:
The name is misleading. Just call this `rootTiledOp`  or something. `lastTiledProducer..` sounds like the last producer that is fused in.